client/components/domains/contact-details-form-fields/custom-form-fieldsets/us-address-fieldset.jsx

```diff
@@ -18,15 +20,15 @@ const UsAddressFieldset = ( props ) => {
 				label={ getStateLabelText( countryCode ) }
 				countryCode={ countryCode }
 				selectText={ STATE_SELECT_TEXT[ countryCode ] }
-				{ ...getFieldProps( 'state', {
+				{ ...getFieldProps?.( 'state', {
```

Comment:
Thank you for doing this the right way ❤️

Reply:
Types forced me hehe.